content/en/blog/2024/ambient-reaches-beta/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Istio’s new sidecar-less ‘Ambient Mode’ reaches Beta in version 1.22"
+description: The latest release from Istio brings service mesh Layer 4 & 7 features to production readiness without sidecars.
+publishdate: 2024-05-13
+attribution: "Lin Sun (Solo.io), for the Istio Steering and Technical Oversight Committees"
+keywords: [Istio Day,Istio,conference,KubeCon,CloudNativeCon]
+---
+
+Istio ambient service mesh was [announced in September 2022](/blog/2022/introducing-ambient-mesh/) as an experimental branch
+that introduced a new data plane mode in Istio that did not require sidecars, offering a simplified operational experience of
+Istio. After 20 months of hard work and collaboration within the Istio community, with contributions from Solo.io, Google,
+Microsoft, Intel, Aviatrix, Huawei, IBM, Red Hat, and others, we are excited to announce that ambient mode has reached Beta in
+version 1.22! The beta release of 1.22 indicates the features of ambient mode are now ready for production workloads
+with appropriate precautions. This is a huge milestone for Istio, bringing both Layer 4 and Layer 7 mesh features to production
+readiness without sidecars.
+
+## Why ambient mode?
+
+We listened to the feedback from Istio users and observed a growing demand for mesh capabilities for their applications but
+found the resource overhead and operational complexity of sidecars hard to overcome. Challenges that Istio sidecar users
+have shared with us include: how Istio can break applications after sidecars are added, the large consumption of resources by
+sidecars, and the inconvenience of the requirement to restart application pods with every new proxy release.
+
+As a community, we listened and designed ambient mode to tackle these problems,
+alleviating the previous barriers of complexity faced by users looking to implement service mesh. This new feature from Istio
+was named 'ambient mode' as it was designed to be transparent to your application, ensuring no additional configuration was
```

Comment:
Should i use double quote here? @dhawton

Reply:
I can see it going either way, personally.

Reply:
Ok cool, thanks. Will submit a PR if others complain!